Update Spanish Translation

diegombeltran opened the pull request:
* Fix previous suggestions
* Adjust changes to PR limit
* Use UTF-8

ihhub commented on files/lang/es.po:
```diff
@@ -16,178 +16,185 @@ msgstr ""
 "Do you want to regain control from AI? The effect will take place only on "
 "the next turn."
-msgstr ""
+msgstr "¿Quieres recuperar el control de la IA? "
```
Do we need a space at the end of the sentence? Also only the first sentence has been translated.

Joarico replied:
Proper translation should be "¿Quieres recuperar el control de la IA? El efecto sólo tendrá efecto en "
                                               "el siguiente turno."
The space was already present on the original string.

ihhub commented on files/lang/es.po:
```diff
@@ -2022,11 +2012,11 @@ msgstr ""
 
 #, fuzzy
```
For this case and at line 2017 `#, fuzzy` comment wasn't removed while for lines below it was. Both variations lead to the same results but it is good to be consistent.

ihhub commented on files/lang/es.po:
```diff
@@ -2694,13 +2679,13 @@ msgid "Damage"
 
 msgid "Hit Points Left"
-msgstr "Vida Restante"
+msgstr "Pts. de Daño Restantes"
```
Could you please check monster info dialog whether there is enough space to keep this long sentence?

diegombeltran replied:
Hi @ihhub , unfortunately, no space left. I'm going to revert.

ihhub reviewed (changes requested):
Hi @diegombeltran , I left few comments here. Could you please take a look at them?

diegombeltran commented:
Good afternoon.

I will check the comments and the conflicts before sunday.

Thank you for your time reviewing!

Joarico commented on files/lang/es.po:
```diff
@@ -16,178 +16,185 @@ msgstr ""
 
 msgid "TRADE"
-msgstr ""
+msgstr "COMERCIAR"
```
If it doesn't fit it could be replace by "CAMBIAR" which is a bit shorter and means Exchange.

Joarico commented on files/lang/es.po:
```diff
@@ -16,178 +16,185 @@ msgstr ""
 
 msgid "X-L"
-msgstr ""
+msgstr "XL"
```
Why remove the dash between X and L in original text? That's not language dependant.

diegombeltran replied:
I think XL is the correct abbreviation. In fact, I would change it in the English language, instead of the Spanish file:
https://www.macmillandictionary.com/thesaurus-category/british/clothing-sizes

Joarico replied:
Then so be it, you seem to be better informed about this.

diegombeltran replied:
@zenseii what do you think?

Joarico commented on files/lang/es.po:
```diff
@@ -16,178 +16,185 @@ msgstr ""
 
 msgid "ALL"
 msgstr ""
 
```
Forgot to translate "ALL" which is "TODO"

diegombeltran replied:
The button is small, it doesn't fit. Until a solution is thought, I wouldn't change it.

Joarico commented on files/lang/es.po:
```diff
@@ -16,178 +16,185 @@ msgstr ""
 
 msgid "SELECT"
-msgstr ""
+msgstr "ESCOGE"
```
zenseii is right, the verb form is not right. However, I would prefer SELECCIONAR if it fits available space, though, because it's more faithful to the original text.

diegombeltran replied:
This was because of the available space. I'm checking all this buttons to see if they fit.

Joarico commented on files/lang/es.po:
```diff
@@ -16,178 +16,185 @@ msgstr ""
 "GAME"
-msgstr ""
+msgstr "ESTÁNDAR"
 
```
Second line not translated, should be "PARTIDA\n"
                                                             "ESTÁNDAR"

diegombeltran replied:
Size

Joarico commented on files/lang/es.po:
```diff
@@ -16,178 +16,185 @@ msgstr ""
 "GAME"
-msgstr ""
+msgstr "CAMPAÑA"
 
```
Second line not translated, should be "PARTIDA\n"
"DE CAMPAÑA"

diegombeltran replied:
Size

diegombeltran replied:
There was another problem, the Ñ symbol was not available when I started to translate this strings. I'm not sure if this is already resolved @zenseii

Joarico commented on files/lang/es.po:
```diff
@@ -16,178 +16,185 @@ msgstr ""
 "CAMPAIGN"
-msgstr ""
+msgstr "ORIGINAL"
 
```
Second line not translated, should be "CAMPAÑA\n"
"ORIGINAL"

diegombeltran replied:
Size

Joarico commented on files/lang/es.po:
```diff
@@ -16,178 +16,185 @@ msgstr ""
-msgstr ""
+msgstr "REGALO"
 
 msgid "guildWell|EXIT"
```
Is "guildWell|EXIT" a proper string from the game? If it's so, which i doubt, the spanish translation has to be modified.

Joarico commented on files/lang/es.po:
```diff
@@ -16,178 +16,185 @@ msgstr ""
 
 msgid "OKAY"
-msgstr ""
+msgstr "OK"
```
"VALE" makes a better translation than using english words, and it takes the same characters as the original text.

diegombeltran replied:
Looks like the RAE says so (Although capital OK is allowed). Changed to "VALE".

Joarico replied:
Thank you for your work Diego, great translation so far!

Joarico commented on files/lang/es.po:
```diff
@@ -16,178 +16,185 @@ msgstr ""
+msgstr "¿Quieres recuperar el control de la IA? "
 
 msgid ""
 "A few\n"
```
Not translated. "Un puñado"

diegombeltran replied:
Not all the strings were translated in this PR. There's a limit of lines to avoid long reviews.

Joarico commented on files/lang/es.po:
```diff
@@ -280,11 +287,10 @@ msgid "army|Legion"
 
 msgid "All %{race} troops +1"
-msgstr "Toda la tropa %{race +1"
+msgstr "Toda la tropa %{race +1}"
```
The proper translation would be "Toda tropa de %{race} +1"

Joarico commented on files/lang/es.po:
```diff
@@ -372,46 +378,43 @@ msgstr "Velocidad"
 msgid "Army Order"
 msgstr ""
```
Missing translation "Orden del Ejército"

Joarico replied:
If it refers to the turn order, "Orden del Turno" would make a better translation.

diegombeltran replied:
For the next PR

Joarico commented on files/lang/es.po:
```diff
@@ -372,46 +378,43 @@ msgstr "Velocidad"
 
 #, fuzzy
 msgid "Auto Spell Casting"
 msgstr "Conjuradores"
```
Proper translation should be "Lanzamiento Automático de Conjuros" or "Auto Conjurar" if space is problem.

diegombeltran replied:
The string reviewed is "On".

Joarico commented on files/lang/es.po:
```diff
@@ -445,9 +448,8 @@ msgstr ""
+msgstr "Salir de este menú."
 
 msgid "Okay"
 msgstr "OK"
```
"Vale" is better translation to avoid using english terms.

diegombeltran replied:
The string reviewed is "Exit this menu"

Joarico commented on files/lang/es.po:
```diff
@@ -520,7 +522,7 @@ msgid "Defense"
+msgstr "Pot. de Hech."
 
 msgid "Knowledge"
 msgstr "Conocimiento"
```
This won't fit. In original Heroes 2 spanish version, they've translated "Conocim." I suggest using "Saber" as translation, which means the same.

diegombeltran replied:
The string reviewed is "Spell Power".
Anyways, "Conocimiento" fits, I tested it.

Joarico commented on files/lang/es.po:
```diff
@@ -520,7 +522,7 @@ msgid "Defense"
 
 msgid "Spell Power"
-msgstr "Pot. de Hech"
+msgstr "Pot. de Hech."
```
In the original Heroes 2 spanish version, they use Pot. Hechizo, but I think "Potencia" is a better solution.

diegombeltran replied:
So be it!

Joarico commented on files/lang/es.po:
```diff
@@ -650,7 +652,7 @@ msgid "View Ballista info"
 msgstr ""
```
missing translation "Ver info de la Balista"

Joarico commented on files/lang/es.po:
```diff
@@ -833,26 +834,23 @@ msgstr "Reflejo creado."
 msgid "The mirror image is destroyed!"
 msgstr ""
```
Missing translation "¡La imagen espejada es destruida!"

Joarico commented on files/lang/es.po:
```diff
@@ -965,22 +963,20 @@ msgstr "Alianza Rota"
-#, fuzzy
 msgid "Dragon Alliance"
-msgstr "Alianza Rota"
+msgstr "Alianza Dragón"
```
"Alianza Dragontina" is a better translation

Joarico commented on files/lang/es.po:
```diff
@@ -2779,35 +2763,34 @@ msgid "Music"
-#, fuzzy
 msgid "MIDI Expansion"
-msgstr "Mansión"
+msgstr "MIDI Extendido"
```
"MIDI Expansión" is the proper transaltion here. It refers to the new midi music for the towns in the expansion The Price of Loyalty, not to an extendend duration of the midi soundtrack.

diegombeltran replied:
Changed

Joarico commented on files/lang/es.po:
```diff
@@ -2816,93 +2799,94 @@ msgid "Resource cost:"
 msgid "Load Game"
-msgstr "Cargar juego"
+msgstr ""
+"Cargar\n"
```
If it's a button the translation it's OK cause otherwise won't fit. If not, refer to what zenseii said.

diegombeltran replied:
I think there are some strings like this one. I'm going to review them all.

Joarico commented on files/lang/es.po:
```diff
@@ -2816,93 +2799,94 @@ msgid "Resource cost:"
+"Partida"
 
 msgid "Start a single or multi-player game."
 msgstr "Comenzar una partida simple o multijugador."
```
"Comenzar una partida individual o multijugador." would make a better translation

diegombeltran replied:
Fixed

Joarico commented on files/lang/es.po:
```diff
@@ -2923,92 +2907,85 @@ msgid ""
-#, fuzzy
 msgid "System Info"
-msgstr "Opciones del Sistema"
+msgstr "Información de Sistema"
```
"Información del Sistema" or "Info. Sistema" if space is problem

diegombeltran replied:
This is debatable, both work, but I'm going to change it to make you happy :D

diegombeltran commented on files/lang/es.po:
```diff
@@ -16,178 +16,185 @@ msgstr ""
 
 msgid "CANCEL"
-msgstr ""
+msgstr "ATRÁS"
```
Again, it was due to the button size

diegombeltran commented on files/lang/es.po:
```diff
@@ -16,178 +16,185 @@ msgstr ""
 "CAMPAIGN"
-msgstr ""
+msgstr "EXPANSION"
 
```
Button size

diegombeltran commented on files/lang/es.po:
```diff
@@ -16,178 +16,185 @@ msgstr ""
 
 msgid "HOT SEAT"
-msgstr ""
+msgstr "EN SITIO"
```
I had doubts with this one. I think "POR TURNOS" leads to confusion (At least for newcomers). "SILLA CALIENTE" would be the direct translation (maybe "ASIENTO CALIENTE" sounds better?). The translation breaks the button due to size in both cases. I would also leave "HOT SEAT" (as I would leave "OK" as confirm button), but I understand some people prefer to literally translate everything. My vote goes for "ASIENTO CALIENTE".

Joarico replied:
Asiento Caliente it's OK for me also, especially if it fits the button hehe

diegombeltran commented:
Good evening all,

I updated and fixed some of the strings you mentioned. There are some buttons that I can't change to avoid breaking buttons. Is there a solution planned for this?

diegombeltran commented:
> Hi, @diegombeltran. I've reviewed your PR and found some things you might want to look at if you have time.
> 
> In addition your PR now has 415 changes (total additions and removals). Could you please reduce this to 400 to stay within our limit for translation PRs?

Should I reduce? Maybe we can discard the problematic buttons until a better way comes up.
